optimum/habana/transformers/generation/configuration_utils.py

```diff
@@ -21,3 +21,4 @@ def __init__(self, **kwargs):
 
         self.static_shapes = kwargs.get("static_shapes", None)
         self.ignore_eos = kwargs.get("ignore_eos", None)
+        self.attn_softmax_bf16 = kwargs.get("attn_softmax_bf16", None)
```

Comment:
Could you also add it in the docstring of the class?  You can reuse the help text you wrote in `run_generation.py`.

Reply:
So silly of me that I missed it. Done now.